Source/RevitIFCTools/ProcessPsetDefinition.cs

```diff
@@ -953,7 +953,7 @@ PsetDefinition Process(string schemaVersion, FileInfo PSDfileName, Dictionary<It
 
                string[] applTypeStr = applicableType.Split('/', '.', '=');
                pset.ApplicableType = applTypeStr[0];
-               if (applTypeStr.Count() > 1)
+               if (applTypeStr.Count() > 1 && !pset.ApplicableType.Equals("IfcMaterial", StringComparison.InvariantCultureIgnoreCase))
                   pset.PredefinedType = applTypeStr[applTypeStr.Count() - 1].Replace("\"", "").TrimEnd(',');
```

Comment:
We should probably have this as a function that explains why we are doing this special case check here.